### DISCUSSION TITLE
Add SSD1309 128x128 support / How to create u8g2 pull requests?

### DISCUSSION THREAD

**Discussion by C3002** [olikraus/u8g2#2543] [Q&A]
Hi Oli!
Ich habe eine kleine Erweiterung für ein Display SSD1309 128x128 hinzugefügt, kann aber keinen Branch mit meinen Änderungen erstellen.
Was ist der beste Weg um die Erweiterung in den Master zu bekommen?
Beste Grüße 
CSC

Hi Oli!
I have added a small extension for a Display SSD1309 128x128, but can't create a branch with my changes.
What is the best way to get the extension into the master?
Best regards 
CSC

**Comment by olikraus** [comment]
Hi, am besten macht man die anpassungen an einen fork, d.h. eine kopie von u8g2 in deinem eigenen repo über die github oberfläche, Diesen fork  clonst du dir dann und pflegst die änderungen ein (commit+push). Anschließend erstellst du mir einen pull request von deinem fork auf das orginal (master) u8g2 projekt (ebenfalls von der github oberfläche).

Den fork erstellt man auf dem "code" reiter des master projekts mit dem "fork" button.

English:

1. In the code tab of u8g2 project, use the fork button to create a fork of the u8g2 project in your local repo
2. Clone the u8g2 fork, update the same, do a commit and push
3. In your local repo for the local u8g2 fork, create a pull request for u8g2 master project with your updates

The more detailed instrutions are here: https://gist.github.com/Chaser324/ce0505fbed06b947d962

Especially, you may need to resync your local fork before creating the pull request.
